go/libkb/env.go

```diff
@@ -1040,8 +1040,29 @@ func (e *Env) GetMountDir() (string, error) {
 	}
 }
 
+func ParseAppStartMode(s string) AppStartMode {
```

Comment:
Would be nice to return error if the string was invalid.

I guess in the past we'd have an enum for the error case.. e.g. AppStartModeError.